gobblin-core-base/src/main/java/org/apache/gobblin/source/extractor/extract/FlushingExtractor.java

```diff
@@ -88,6 +90,9 @@ public abstract class FlushingExtractor<S, D> extends EventBasedExtractor<S, D>
   public static final String WATERMARK_COMMIT_TIME_METRIC = "state.store.metrics.watermarkCommitTime";
   public static final String COMMIT_STEP_METRIC_PREFIX = "commit.step.";
 
+  public static final String WRITER_OUTPUT_DIR_UPDATE_ENABLED =
```

Comment:
I don't think this name in isolation properly describes what the feature flag enables. I.e. `extractor.writerOutputDirUpdateEnabled` implies we are updating the output dir. But why are we updating it? And it also seems to imply it may work for all extractors, but this is specific to the flushing extractor.

The goal is to give a unique output directory based on the current task attempt id. So the config name should give a hint to that

Reply:
changed config name to ENABLE_UNIQUE_WRITER_OUTPUT_DIR_WITH_TASK_ATTEMPT_ID, as well as included a comment block for the config